app/api/users.py

```
@@ -0,0 +1,67 @@
+    )
+    if not user:
+        raise HTTPException(status_code=401, detail="Incorrect email or password")
+    access_token = user_service.create_access_token(
+        {"sub": str(user.id), "email": user.email}
+    )
+    return {"access_token": access_token, "token_type": "bearer"}
+
+
+async def get_current_user(
+    token: Annotated[str, Depends(oauth2_scheme)],
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)